tests/testthat/test-estimate_time_varying.R

```diff
@@ -1,13 +1,8 @@
 #### Tests for the static CFR function cfr_static() ####
 # prepare data and common testing elements
 
-# read epidist for EVD onset to death from {epiparameter}
-onset_to_death_ebola <- epiparameter::epidist_db(
-  disease = "Ebola Virus Disease",
-  epi_dist = "onset_to_death",
-  author = "The-Ebola-Outbreak-Epidemiology-Team",
-  single_epidist = TRUE
-)
+# Ebola onset to death distribution comes from Barry et al. 2018
+# a gamma distribution with k = 2.40, theta = 3.33
```

Comment:
Probably easier to keep the parameter naming in comments consistent with base R rather than the traditional mathematical notation.

Reply:
Fixed in https://github.com/epiverse-trace/cfr/pull/88/commits/95613171904caf9380ff8670bdfeaf002059e6c2